### DISCUSSION TITLE
Where does the metric "gha_assigned_jobs" come from

### DISCUSSION THREAD

**Discussion by abuechler** [actions/actions-runner-controller#3088] [Questions]
The Grafana dashboard makes use of a metric gha_assigned_jobs, where does it come from? I can't see it under the metrics of our controller (v0.6.1).

https://github.com/search?q=repo%3Aactions%2Factions-runner-controller%20gha_assigned_jobs&type=code

**Comment by nikola-jokic** [comment]
Metrics are exposed both by the listener and by the controller. The metric you are referring to (`gha_assigned_jobs`) comes from the [listener](https://github.com/actions/actions-runner-controller/blob/7793e1974a0fc6c4dd346b2492ab82a1531593c3/cmd/githubrunnerscalesetlistener/metrics.go#L73)
